docs/user-guide/cli-using-integrating-apiml.md

```diff
@@ -40,53 +55,101 @@ Issue the following command:
 
```

Comment:
Change to:
Logging out causes the API ML token to expire and removes the token from your base profile.
Delete: 
Issue the following command:
Add:
To log out of the API ML: